ipsec_accel/ipsec_accel.py

```diff
@@ -0,0 +1,1103 @@
+#!/usr/bin/python
+#
+# Copyright 2022-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+# Python tool to setup Linux Networking with OVS Offload on Intel® Infrastructure Processing Unit (Intel® IPU)
+
+import sys,argparse
+import os
+
+# Add the parent directory to the system path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from common.utils import *
+
+def build_p4rt_config(test_setup = None):
+
+    if test_setup == None:
+        print("Unable to parse the config.yaml to generate test configuration")
+        sys.exit()
+
+    host_idpf_intf= test_setup.test_config['test_params']['idpf_interface']
+    vf_list = test_setup.test_config['test_params']['vf_interfaces']
+    acc_pr_list = test_setup.test_config['test_params']['acc_pr_interfaces']
+    vm_ip_list = test_setup.test_config['test_params']['ip_list']
+    path = test_setup.test_config['test_params']['host_path']
+    comm_ip_host = test_setup.test_config['test_params']['comm_ip_host']
+    comm_ip_acc = test_setup.test_config['test_params']['comm_ip_acc']
+    ipsec_host = test_setup.test_config['test_params']['ipsec_host']
+    local_vxlan_tunnel_mac =  test_setup.test_config['test_params']['local_vxlan_tunnel_mac']
+    remote_vxlan_ip = test_setup.test_config['test_params']['remote_vxlan_ip']
+    remote_vxlan_mac = test_setup.test_config['test_params']['remote_vxlan_mac']
+
+    print("---------- Generating Configs to run infrap4d, p4rt, OVS and IPsec ----------")
+
+    if len(vf_list) != len(acc_pr_list):
+        print("ERROR: number of vfs and ACC PRs should be the same")
+        return None
+
+    host_command_list = []
+    cmd = f"mkdir -p {path}"
+    host_command_list.append(cmd)
+
+    file = f'{path}/1_host_idpf.sh'
+    host_idpf = 'cat <<EOF > ./'+file+'''
+#!/bin/sh
+#Load the driver
+echo "Load the IDPF Driver on the Host"
+modprobe idpf
+sleep 4
+
+echo ""
+echo "Check the Interfaces are up"
```

Comment:
Update language to describing what the verb action more clearly.

```suggestion
echo "Checking if the interfaces are up"
```